src/Assets/ugui-mvvm/INPCBinding.cs

```diff
@@ -290,6 +290,7 @@ public void ClearHandlers()
 
         [SerializeField]
         ComponentPath _view;
+        public ComponentPath View => _view;
```

Comment:
Why expose this public property?

Reply:
I'm exposing these properties for a two reasons:
- We are already exposing this as a property on CommandBindings, so having each of these types  of property bindings have the same publicly accessible surface area makes the most sense to me.
- This allows users to write utilities which understand and collect information on the component paths, which means that we need these exposed at least internally.  For example I'm planning on adding a utility for our project which validates all bindings in a scene.

Reply:
Sounds reasonable